README.md

```diff
@@ -114,7 +114,7 @@ Install this project with:
 Change the main branch to the right one, execute the following command:
 
```

Comment:
- [ ] [optional] On your key future section, the desired features are functions the user can interact with eg add a score view a score